### ISSUE TITLE
How to handle directline offline in newer version

### ISSUE THREAD

**Issue by Ant01ine** [NGRP/node-red-contrib-viseo#146]
Hello Viseo team,

First off, thanks for all the good work!
I'm confused about how to interact with a local bot using the latest versions of the Viseo framework and MS bot builder node.

To avoid going through to Azure in a development environment, we've been using https://github.com/ryanvolum/offline-directline, so that our frontend can connect to a local directline pointing to a local Viseo instance. However as stated in this issue https://stackoverflow.com/questions/55812986/how-can-i-use-offline-directline-for-microsofts-botframework-v4 directline-offline is not compatible with Bot framework v4.

I see in recent versions of Viseo, that you are upgrading to bot framework v4 as well. How would you recommend emulating Azure without using directline-offline ?

Also, if I'm not mistaken, the API URL is no longer /api/v1/messages but /server-botbuilder. Am I correct or am I missing something ?

Thank you,

Antoine

**Comment by JpEncausse** [comment]
Yes the API URL has change because we cleanup all the "connector" for v1.0

We no longer use the DirectLine Offline because it is incomplete there is a lot of missing elements (Cards, Image, Uploads,etc ...) and the authentication is not implemented.

We'll dig into it when one of our customer will need a workarouind Azure Bot Registration
